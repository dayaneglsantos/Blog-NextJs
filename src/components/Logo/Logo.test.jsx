import { render, screen } from '@testing-library/react'
import LogoLink from '.'

describe('<LogoLink />', () => {
  it('should render', () => {
    render(<LogoLink url="local" altText="text" />)
    const image = screen.getByRole('img')
    expect(image).toHaveAttribute('alt', 'text')
  })
  it('should match snapshot', () => {
    const { container } = render(<LogoLink srcImg="local" altText="text" />)
    expect(container).toMatchSnapshot()
  })
})
