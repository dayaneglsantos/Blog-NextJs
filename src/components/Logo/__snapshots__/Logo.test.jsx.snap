// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoLink /> should match snapshot 1`] = `
.c0 {
  text-decoration: none;
  text-align: center;
}

.c1 {
  height: 10rem;
  width: 10rem;
  border-radius: 50%;
  margin: 0 auto;
  object-fit: cover;
}

<div>
  <div
    class="c0"
  >
    <img
      alt="text"
      class="c1"
    />
  </div>
</div>
`;
